content/2017-munich/sponsor.md

```diff
@@ -221,7 +306,7 @@ Thanks for your interest in sponsoring PromCon!
   loss to sponsors' properties by fire, theft, accident or any other cause
   whether the result of negligence or otherwise or case of force majeure.
 * Refund Policy - Please contact
-  [billing@robustperception.io](mailto:billing@robustperception.io) regarding
+  [Julius Volz](mailto:julius.volz@gmail.com) regarding
```

Comment:
Yeah, was wondering whether we should use the CNCF disbursements list for this or create a separate one. Probably that one is fine?